plugins/janus_recordplay.c

```diff
@@ -615,6 +621,13 @@ static const char *janus_recordplay_parse_codec(const char *dir, const char *fil
 					return NULL;
 				}
 				const char *c = json_string_value(codec);
+				if (data) {
+					/* Found! */
+					c = !strcasecmp(c, "text") ? "text" : "binary";
```

Comment:
Just put "text" here: as I said in a previous comment, we currently don't support saving binary data in mjr yet anyway. That will be done in another effort later on, so I'll take care of updating the code here when that happens.

Reply:
Agreed!